### DISCUSSION TITLE
Use as Relay Service for another smtp server

### DISCUSSION THREAD

**Discussion by ReptoxX** [postalserver/postal#2646] [Help with using Postal]
Hello together, 

i've read through a couple of posts here but i'm still unsure if my desired workflow would work.
We currently use a provider for sending mass mails through smtp. As this is somewhat unreliable we've been searching for a solution with auto-resend functionality and an easy to use HTTP Endpoint for sending mails.
Handling bounces is obviously a bonus i'm happy about :)

As we're sending mass mails, we can't switch our outgoing server-provider and don't want to entirely rely on postal. But I would love to use it as an interface.

So is it possible to configure postal to accept (outgoing) mails, send them to our other smtp server and receive the bounce messages sending out http webhooks?

Regards,
ReptoxX

**Comment by willpower232** [comment]
You can configure email leaving Postal to go to a relay in the config file but it cannot use authentication which may or may not be a problem for you https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/config/postal.defaults.yml#L79-L83

You would have to configure your relay to send the bounces back to Postal and then you could further handle them with a route if you wish https://docs.postalserver.io/other/auto-responders-and-bounces

**Reply by ReptoxX** [reply]
It's great news! Our Provider doesn't require authentication, just pure IP Validation.
For the bounces, can't we just set another E-Mail as return-path and set the mx server of that domain to the postal system? Shouldn't that be sufficient?

**Reply by willpower232** [reply]
I think others have asked this in the past, I'm sure it is slightly possible if you create the whole SMTP message yourself and don't use the API.

**Comment by ReptoxX** [comment]
Hello again,

is there an option to disable the "UnauthenticatedFromAddress" validation?
And is by any chance there an option to disable DKIM for a certain domain?

Regards

**Reply by willpower232** [reply]
You may want to look in to the sender header if you need to include a different email address. Unfortunately Postal will only let the email be sent if API key matches a domain it already knows about (and you're using an API key from the same server/organisation)

I don't think it can be disabled but I'm not sure if it applies if you don't create the records.

**Reply by ReptoxX** [reply]
It makes sense, don't get me wrong, but it's just unfortunate because we don't have an api yet to dynamically add domains.
For us both systems would work, enabling the api to be able to add domains or disable the check.
I will add an api service for our purpose to the stack, but it should be included. I'm adding my "+1" to the api thread.

Unfortunately it applies even without the proper dns records. That would be an improvement to check whether or not the dns is setup correctly before adding dkim to the mail.
